docs/items/items.md

```diff
@@ -30,4 +30,33 @@ Registering an Item
 
 Items must be [registered][registering] to function.
 
-[registering]: ../concepts/registries.md#registering-things
+Coloring an Item
+----------------
+
+There are items Minecraft prefers to render by coloring one single texture. Leather helmets, spawn eggs, potions and other.
+
+### Item Color Handler
+
+Item color handler is just an instance of `IItemColor`. The interface contains only one method:
+
+```java
+getColorFromItemstack(
+  ItemStack stack, 
+  int tintIndex)
+```
+
+It provides ItemStack of item that is being painted and the tint index. 
+
+When coloring an ItemBlock, the tint index is the same thing as it is with blocks, but if an Item is being painted, the tint index value indicates which layer of an item is being painted. Layers of item are specified in its model JSON file and are layers of its texture.
+
+### Registering an Item Color Handler
```

Comment:
I don't think either of the `!!! important`s need to be `!!! important` here.

Instead, I would have the description be:
"Item color handlers are registered by calling `ItemColors#registerItemColorHandler(IItemColor, Item)`. An `ItemColors` instance can be obtained by calling `Minecraft#getItemColors()()`.

This must be done during the initialization phase, and on the client side only."